modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/FlashClientCodegen.java

```diff
@@ -81,7 +86,8 @@ public FlashClientCodegen() {
         importMapping.put("File", "flash.filesystem.File");
 
         // from
-        setReservedWordsLowerCase(Arrays.asList("add", "for", "lt", "tellTarget", "and",
+        // TODO Are those case sensitive or insensitive
+        super.registerReservedWordsCaseInsensitive(Arrays.asList("add", "for", "lt", "tellTarget", "and",
```

Comment:
As far as I remember(I was ActionScript3 developer long time ago and unfortunately don't have Flash Builder right now to check). AS1 was case insensitive, but AS2 and AS3 are case sensitive and highly typed languages. I don't know what version of ActionScript this generator targeted, but assume that AS3.

Reply:
Then I will let them to be case insensitive just to be sure, does that sound ok?